services/movie.py

```diff
@@ -1,14 +1,17 @@
 from django.db.models import QuerySet
-
+from django.db import transaction
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
Change to Optional[list[int]]
Because by default it accepts None
Check and do the same in other places